api-tester/src/main/java/com/revenuecat/apitester/kotlin/StoreProductAPI.kt

```diff
@@ -19,7 +20,7 @@ private class StoreProductAPI {
             val price: Price? = price
             val title: String = title
             val description: String = description
-            val subscriptionPeriod: String? = subscriptionPeriod
+            val period: Period? = period
```

Comment:
kinda feels like overkill, but might be good to have a PeriodAPI as well to test the values within there

Reply:
I think we should add that. All public API should be API tested